DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<p>"It is composed of several independent runs of the model." -- or something to that effect</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>